posttroll/backends/zmq/ns.py

```diff
@@ -15,39 +16,55 @@
 nslock = Lock()
 
 
-def zmq_get_pub_address(name, timeout=10, nameserver="localhost"):
+def zmq_get_pub_address(name:str, timeout:float|int=10, nameserver:str="localhost"):
```

Comment:
I'm not at all familiar with type hinting syntax, but this looks different to the examples I see on the net or in Satpy
* whitespace between the variable and type `name: str`
* whitespace between possible types and kwarg default values: `timetout: float | int = 1`

Reply:
addressed in [18b6c1a](https://github.com/pytroll/posttroll/pull/72/commits/18b6c1af9277d306a8640a8b674861b68dec915d)